sklearn/neighbors/classification.py

```diff
@@ -81,6 +81,27 @@ class KNeighborsClassifier(NeighborsBase, KNeighborsMixin,
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
         Doesn't affect :meth:`fit` method.
+        
+    Attributes
+    ----------
+    classes_ : array of shape = (n_classes,)
+        Class labels known to the classifier
+        
+    effective_metric_ : string or callble
+        The distance metric used. It will be same as
```

Comment:
Please try to use 79 characters per line.

Reply:
Thanks @thomasjpfan please review the updated version.